tests/compose-tests/libbasic-test.sh

```diff
@@ -0,0 +1,41 @@
+# This used to live in test-basic.sh, but it's now shared with test-basic-unified.sh
```

Comment:
Minor: any reason for formatting this as a lib? Seems cleaner as a `basic-test.sh` we just source directly like ostree's `basic-test.sh`. Not super opposed either way though.

Reply:
I kept this as just to avoid too much churn but may revisit later.